src/helia-server.ts

```diff
@@ -251,12 +251,27 @@ export class HeliaServer {
 
     const opController = new AbortController()
     setMaxListeners(Infinity, opController.signal)
-    request.raw.on('close', () => {
-      if (request.raw.aborted) {
-        this.log('request aborted by client')
-        opController.abort()
+    const cleanupFn = (): void => {
+      if (request.raw.readableAborted) {
```

Comment:
`request.raw.aborted` is deprecated. message says to use `readable.message.aborted` but we don't have access to that?